src/centos/scripts/start-marklogic.sh

```diff
@@ -13,6 +13,41 @@
 
 cd ~
 
+################################################################
+# Read in ENV values
+################################################################
+
+# If an ENV value exists in a list Append it the /etc/marklogic.conf file 
+if [[ -z $OVERWRITE_ML_CONF ]] || [[ "$OVERWRITE_ML_CONF" = "false" ]] ; then
+    echo "Not writing to /etc/marklogic.conf"
+else
+    echo "Deleting previous /etc/marklogic.conf ,if it exists, and overwriting with env variables"
+    rm -f /etc/marklogic.conf
+    sudo touch /etc/marklogic.conf && sudo chmod 777 /etc/marklogic.conf
```

Comment:
I had permission issues when testing this and gave it something permissive, I couldn't find anywhere what the least permissions I should give in order to write to the file, if anyone feel free to comment.

Reply:
According to https://docs.marklogic.com/guide/ec2/GettingStarted#id_64949 this file is only read by MarkLogic on start up so perhaps we could leave it as read-only.

Reply:
Updated: 66c8142208675f7ab08dcf4250bcb26c15ec2f12